static/templates/invitation_failed_error.hbs

```diff
@@ -16,3 +16,14 @@
     <p id="invitation_non_admin_message">{{t "Organization administrators can reactivate deactivated users." }}</p>
     {{/if}}
 {{/if}}
+{{#if license_limit_reached}}
+    {{#if has_billing_access}}
+        {{#tr}}
+        To invite users, please <a href="/billing/#settings">increase the number of licenses</a> or <a href="/help/deactivate-or-reactivate-a-user">deactivate inactive users</a>.
+        {{/tr}}
+    {{else}}
+        {{#tr}}
+        Please ask a billing administrator to <a href="/billing/#settings">increase the number of licenses</a> or <a href="/help/deactivate-or-reactivate-a-user">deactivate inactive users</a>, and try again.
```

Comment:
These should use the `z-link` technique to avoid including the link syntax in the string sent to translators.